src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -606,16 +592,16 @@ private static Route getMaxSeaRoute(final GameData data, final Territory start,
    * Removes the inner circle neighbors
    * neutral - whether to include neutral countries.
```

Comment:
* Would recommend changing "exactly a certain # of territories away" in the summary to "exactly one territory away".
* Description of `neutral` parameter can be removed.

Reply:
Good suggestion, shortened it a good bit in: c416e01c54889ce28eb4a7a78017df8637abb2c3